### ISSUE TITLE
Header text for "discussion" and "edit" and "view pdf" are not accessible / 508 complaint

### ISSUE THREAD

**Issue by NoahKunin** [ombegov/a130#9]
The color contrast (hyperlink blue on the dark blue header) I can barely make them out, and a quick check indicates that it doesn't path either the AA or AAA WCAG standards. 

http://www.w3.org/WAI/intro/wcag

Example simulating WCAG standards showing the menu items vanish:

<img width="462" alt="screen shot 2015-10-21 at 2 20 13 pm" src="https://cloud.githubusercontent.com/assets/565071/10650557/f5474f3e-77fe-11e5-81ec-26fadc4d84fa.png">

Any place without an outline is a problem. This makes it difficult to impossible for members of the public with **impaired ability to see the color spectrum to know where to participate.**

Simply making the links white should fix it. Re-scanned with a quickie CSS change:

<img width="342" alt="screen shot 2015-10-21 at 2 27 22 pm" src="https://cloud.githubusercontent.com/assets/565071/10650759/0ab684a6-7800-11e5-8744-82c47bb1354e.png">

Future CIO.gov products should follow the agreed upon CIO [US Web Design Standards](https://playbook.cio.gov/designstandards/visual-style/#colors) in general, and the section on [color specifically](https://playbook.cio.gov/designstandards/visual-style/#colors).

**Comment by theplaz** [comment]
Agreed!  Should have been caught in QA!

**Comment by jlberryhill** [comment]
Thanks, @theplaz. Those links were a bit of a last-minute add. Will work on getting them compliant. cc @justgrimes

**Comment by rebeccawilliams** [comment]
Thank you! This was fixed with #14 & #15.

**Comment by NoahKunin** [comment]
:+1:
